test/link/test.sh

```diff
@@ -151,6 +151,14 @@ rc=$(($? || $rc))
 tryCmp overlay/out.gb $gbtemp
 rc=$(($? || $rc))
 
+i="sizeof-startof.asm"
```

Comment:
Why does this test have to be special-cased?

Reply:
Turns out it doesn't.